custom_components/inpost_air/api.py

```diff
@@ -73,6 +73,15 @@ async def search_parcel_locker(self, locker_code: str) -> dict[str:Any] | None:
 
         return parcel_locker
 
+    async def get_parcel_lockers_list(self) -> list[dict[str:Any]]:
+        """Get parcel lockers list."""
+        response = await self._request(
+            method="get", url="https://inpost.pl/sites/default/files/points.json"
```

Comment:
Should't we use points from greencity.pl ?

Reply:
There are the same endpoints in greencity.pl and inpost.pl and all other endpoints that we use are hitting inpost.pl so for consistency this is also using inpost.pl